### PR TITLE
Add transaction flow

### PR DESCRIPTION
**Pull request by CallumD**
Screenshot 2020-03-16 at 10 50 34

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.5.0
```
Comment (sztupy) [inline .ruby-version]: Please use 2.6.5

File: img/transaction_flow/api-flow-diagram [hunk 2]
```diff
@@ -0,0 +1 @@
...uKTG42Uiwt2HX5qggZp+3hA8orbbTU1ztcdJDbGn6Db+fmA0mCWoU91d8kpYsW4EApQta6lYvYBJCIh5OcQpHsXj2zkiu5bUP2BkojttZ6fj2zzlt6n/EPMLe6DBSGqEEDuaRwcFRue43+FF4rrOd57KMMceT3HPzML0fX7XtHhtjyBIn83X25iByFEF3Q06GN5vrAyW8K/6oSHVtxJDtjhCyPdqyiLro9SvOUcT6HKyznDZD6E8csYrSABPCTo+UNrs0jswR+xNE385rP4LCZ5ypQ/zOqR08ORnXdfoeeuCMc+qBLBu27JuNbc5U8PvJdTu15QNx3dPYKDGFlm84se44+XSuusL0gajlzs0Ihnfg6YBH0JweMaTzopJ2x1F6R6WSjhqMeaQAYupr9k+9o5VTRFgf1RnFVA6LA7WKNLvqAAV9DiNcPvM++KStT80nXTXX/cwthmv3tBjTbm91u4g8z9SjfCltzYDI7StEWsu4TLPzTFJMEsCaKoaQaopAfi3gAsweIov530wWGaUyRPuqF0ue4rej/13NHew6VUmkwdE5eU3KM07yHCYrthwgtrFndNDzRsIh/5Ihcg4PsWz753TNDHti/RQxmVYGXSf49TVOPUZyEvslL1EZescBg37H2I92vqB/KgeAV9sGS0ep3jPIqgxBtqzUaEAg9c2eQDop9Ra9bKeY4oRilDgsWdfY59LrInSVgBQsYAlhb0jWDnGQbcjWZhnZjQCYFsV0toq8bRRqlN5olPdDQudco5+ioO06Y84xlXs6IcXXZr1StrMlWMFCJnK4W+tr0/B5nceIiVFRHgLy/TPzz/MiRHOpO5shopS+t29ey7hom0MdBSP57I7ZwaKsoyKFasw7hVDdLfBbcUblcGIIwrA4hIvKWgRBTil1xiVRnBXOTrHXYMrOi3lVZfF5CQuC3fiDJz9F8Q2iOLkgnsZRYW8yWYemi4vffdLF2cUXSBAdbRZuGfxYyb6+h3mgydwUkpaAcd8Ecubwbq1o5Nj27FRWOXYe2HCmaW3+xEcKyac5GMoBXmNLmAqUj2ixgMWB7bWLy45jU412Cp8v2jtDqwXPDxHxne1cVnEq521fQan2CBxt8cRXQ2G/rUJ+3mNj19ok+z1X7I8VneF2T/GCHFcRea0jxFXoZf3HIbls1H9h07r7Cw==</diagram></mxfile>
```
Comment (sztupy) [inline img/transaction_flow/api-flow-diagram]: In the docs we started proposing the use of the createAndFund one-call api, and enforce the usage of external IDs for both senders and transactions. We're also having a chat to force-enable auto cancel to any new MTO customer and making it the default. We'll also need to have a look at our current cancel API as it's awful

File: .ruby-version (hunk 1 above)
Comment (CallumD) [reply .ruby-version]: Agreed. 2.5.0 was in the gemfile.lock hence keeping it that way initially. Have pushed this to 2.6.5

File: img/transaction_flow/api-flow-diagram (hunk 2 above)
Comment (CallumD) [reply img/transaction_flow/api-flow-diagram]: Yeah totally agree just wanted to make sure this diagram got in so we can iterate on this.

**Pull request merged**